docs/extending_syzkaller.md

```diff
@@ -0,0 +1,39 @@
+# Extending syzkaller
```

Comment:
`Extending syzkaller` is a very generic name for the file and doc.
Please rename file and title to include "new OS" in some form, e.g. "adding_new_os_support.md" or "new_os_support.md" or something along these lines.

Reply:
Done, thank you! I made the changes you suggested, let me know what you think :)